NYPL/sfr-bookfinder-front-end
# v0.8.5 Improving Read Online experience and tracking

[Jira Ticket](http://jira.nypl.org/browse/SFR-885)

### This PR does the following:
- Adds Google Analytics Event tracking for the Read Online and Download options
- Fixes protocol (HTTP/HTTPS) issues for the local and development environments, providing a smoother experience for development and testing

### Testing requirements & instructions: 
-  Verify that the `local` and `development` branches load ePub files using the relevant webpub-viewer instance
- Check the Google Analytics dashboard to confirm that events are being generated
